src/Microsoft.TemplateEngine.TemplateLocalizer/Commands/Export/ExportCommand.cs

```diff
@@ -11,36 +12,39 @@ internal sealed class ExportCommand : ModelBoundExecutableCommand<ExportCommandA
     {
         private const string CommandName = "export";
 
-        public ExportCommand(ILoggerFactory loggerFactory) : base(loggerFactory) { }
+        private Argument<IEnumerable<string>> _templatePathArgument = new Argument<IEnumerable<string>>("template-path")
+        {
+            Arity = ArgumentArity.OneOrMore,
+            Description = LocalizableStrings.command_export_help_templatePath_description,
+        };
 
-        public override Command CreateCommand()
+        private Option<IEnumerable<string>> _languageOption = new Option<IEnumerable<string>>("-l")
         {
-            Command exportCommand = new(CommandName, LocalizableStrings.command_export_help_description);
-            exportCommand.AddArgument(new Argument<string>("template-path")
-            {
-                Arity = ArgumentArity.OneOrMore,
-                Description = LocalizableStrings.command_export_help_templatePath_description,
-            });
-            exportCommand.AddOption(new Option("-r")
-            {
-                Name = "--recursive",
-                Description = LocalizableStrings.command_export_help_recursive_description,
-            });
-            exportCommand.AddOption(new Option<string>("-l")
-            {
-                Name = "--language",
-                Description = LocalizableStrings.command_export_help_language_description,
-                Arity = ArgumentArity.OneOrMore,
-                AllowMultipleArgumentsPerToken = true,
-            });
-            exportCommand.AddOption(new Option("-d")
-            {
-                Name = "--dry-run",
-                Description = LocalizableStrings.command_export_help_dryrun_description,
-            });
-            exportCommand.Handler = this;
+            Name = "--language",
+            Description = LocalizableStrings.command_export_help_language_description,
+            Arity = ArgumentArity.OneOrMore,
+            AllowMultipleArgumentsPerToken = true,
+        };
 
-            return exportCommand;
+        private Option<bool> _recursiveOption = new Option<bool>("-r")
+        {
+            Name = "--recursive",
+            Description = LocalizableStrings.command_export_help_recursive_description,
+        };
+
+        private Option<bool> _dryRunOption = new Option<bool>("-d")
```

Comment:
This wasn't a `bool`, but now it is. Any reason for this change? Same for `--recursive`.

Will this still work if I type `dotnet-template-localizer -d -r` or do we have to type `dotnet-template-localizer -d true -r true` now?

Reply:
`dotnet-template-localizer -d -r` will work. Boolean options allow no value. 
Reason for change: `Option` should be casted when binding if type is not specified, which mismatches handled additionally.